SQL/1.CreateUser.sql

```diff
@@ -0,0 +1 @@
+CREATE USER test_user SUPERUSER LOGIN PASSWORD 'abc';
```

Comment:
Ten user jest do czegoś używany?
Czy to taki template na przyszłość?

Reply:
podobna sytuacja jak z twoim loggerem, po prostu przykład, który i tak będzie poprawiony w branchu ConfigureDB

Reply:
generalnie ConfigureDB zmienia / rozszerza samą konfigurację bazy, to co jest tutaj to raczej "śmieci" które pozwalały sprawdzić, czy zadziała tak jak myślę i zostawiłem je, żeby był jakiś przykład, jakbym zapomniał "jak to miało być"

Reply:
Spoko, dla mnie git